<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConfig</class>
 <widget class="QDialog" name="frmConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>506</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>60</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Settings - qiptables</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Home Network Shell Script</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edtShellHome">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnShellHome">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnEdtShellHome">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="btnRebuildHome">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Public Network Shell Script</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edtShellPublic">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnShellPublic">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnEdtShellPublic">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btnRebuildPublic">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>38</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>IP Tables Home</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edtIpTablesHome">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btnIpTablesHome">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btnEdtIpTablesHome">
       <property name="text">
        <string>View</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <spacer name="horizontalSpacer_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>IP Tables Public</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="edtIpTablesPublic">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnIpTablesPublic">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btnEdtIpTablesPublic">
       <property name="text">
        <string>View</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <spacer name="horizontalSpacer_14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>IP Tables Default</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="edtIpTablesDefault">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="btnIpTablesDefault">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="btnEdtIpTablesDefault">
       <property name="text">
        <string>View</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <spacer name="horizontalSpacer_15">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>/etc/init.d</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="edtInitIpTables">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="btnInitIpTables">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="btnEdtInitIpTables">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <spacer name="horizontalSpacer_16">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblEdit">
       <property name="text">
        <string>File being edited</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="edtCurrentFile">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>278</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="txtFile">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>frmConfig</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnShellHome</sender>
   <signal>clicked()</signal>
   <receiver>frmConfig</receiver>
   <slot>openFileDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnShellPublic</sender>
   <signal>clicked()</signal>
   <receiver>frmConfig</receiver>
   <slot>openFileDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnIpTablesHome</sender>
   <signal>clicked()</signal>
   <receiver>frmConfig</receiver>
   <slot>openFileDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnIpTablesPublic</sender>
   <signal>clicked()</signal>
   <receiver>frmConfig</receiver>
   <slot>openFileDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnInitIpTables</sender>
   <signal>clicked()</signal>
   <receiver>frmConfig</receiver>
   <slot>openFileDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEdtIpTablesHome</sender>
   <signal>clicked()</signal>
   <receiver>frmConfig</receiver>
   <slot>editFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEdtIpTablesPublic</sender>
   <signal>clicked()</signal>
   <receiver>frmConfig</receiver>
   <slot>editFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEdtShellHome</sender>
   <signal>clicked()</signal>
   <receiver>frmConfig</receiver>
   <slot>editFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEdtShellPublic</sender>
   <signal>clicked()</signal>
   <receiver>frmConfig</receiver>
   <slot>editFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEdtInitIpTables</sender>
   <signal>clicked()</signal>
   <receiver>frmConfig</receiver>
   <slot>editFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtFile</sender>
   <signal>textChanged()</signal>
   <receiver>frmConfig</receiver>
   <slot>btnSaveEnable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>frmConfig</receiver>
   <slot>saveText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnIpTablesDefault</sender>
   <signal>clicked()</signal>
   <receiver>frmConfig</receiver>
   <slot>openFileDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEdtIpTablesDefault</sender>
   <signal>clicked()</signal>
   <receiver>frmConfig</receiver>
   <slot>editFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRebuildHome</sender>
   <signal>clicked()</signal>
   <receiver>frmConfig</receiver>
   <slot>rebuildFirewall()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRebuildPublic</sender>
   <signal>clicked()</signal>
   <receiver>frmConfig</receiver>
   <slot>rebuildFirewall()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openFileDialog()</slot>
  <slot>editFile()</slot>
  <slot>btnSaveEnable()</slot>
  <slot>saveText()</slot>
  <slot>rebuildFirewall()</slot>
 </slots>
</ui>
